### ISSUE TITLE
Why hasn't the latest version been released?

### ISSUE THREAD

**Issue by zhengweisk** [madMAx43v3r/chia-gigahorse#210]
chia-gigahorse why new version ? 

Why hasn't the latest version been released?

Because the latest version of the community has already reached 2.0.1

**Comment by Kruger1981** [comment]
Possibly @madMAx43v3r  is busy working to bring us more compression on the plots.
Although the latest Chia UI version has some nice improvements, if you ask me, more compression on the plots seems a bit more important.
But this is just my assumption...

**Comment by madMAx43v3r** [comment]
Yes correct, working on new plot format

**Comment by DudSeb** [comment]
> Yes correct, working on new plot format

Nice, thank you very much! could you reveal some details ? :)

**Comment by Walrusbonzo** [comment]
Looking forward to this new format!

**Comment by iJeys** [comment]
> Yes correct, working on new plot format

Hi Max, Are there approximate dates ? Thanks

**Comment by madMAx43v3r** [comment]
some time next week

**Comment by scoville07** [comment]
@madMAx43v3r sorry for the repeating question, but will there be any impact sooner than later with the new chia 2.1.0 being launched and gigahorse on 1.8.2?

"Note: Version 2.1.0 includes additional changes originally intended for the 2.0.0 hard fork and is a required update."

Thanks for your efforts!

**Comment by iJeys** [comment]
> @madMAx43v3r sorry for the repeating question, but will there be any impact sooner than later with the new chia 2.1.0 being launched and gigahorse on 1.8.2?
> 
> "Note: Version 2.1.0 includes additional changes originally intended for the 2.0.0 hard fork and is a required update."
> 
> Thanks for your efforts!

Don't distract Max, he's working on compressing the plots.

**Comment by madMAx43v3r** [comment]
> "Note: Version 2.1.0 includes additional changes originally intended for the 2.0.0 hard fork and is a required update."

where did you read that?

**Comment by QuintLeo** [comment]
Also note that the "official" CNI version 2.1.0 has a SERIOUS issue with LINUX, as in you can't SEE the client because it blanks out everything inside the client except the "header" section after a couple seconds.
And that issue actually started with the AMD version for 2.0.1-1, NOT with 2.1.0

**Comment by iJeys** [comment]
> Yes correct, working on new plot format

Max how are you doing with the new plotter ? any information ? :)

**Comment by madMAx43v3r** [comment]
a couple of days still

**Comment by Kruger1981** [comment]
@madMAx43v3r 
Not sure if this is a coincidence or not, but since official version 2.1.0 was released yesterday, I started having loads of connection issues and some other random issues.
I am still on GigaHorse 1.8.2 Giga14 and on Chia official GUI 1.8.2

**Comment by strale92** [comment]
@madMAx43v3r
we will be able to use the current c8 compressed plots for the chia farm after the update?
currently there are different opinions in different groups
if you could clarify for us
Thanks

**Comment by madMAx43v3r** [comment]
yes it will be backwards compatible, the new version will be able to farm more C6-C9 also.

**Comment by pdeline06** [comment]
> a couple of days still

Dear Max, well, when already C..? :)

**Comment by iJeys** [comment]
> > a couple of days still
> 
> Dear Max, well, when already C..? :)

Judging by the photo, Max has more important things to do ;)

**Comment by LimakKAAM** [comment]
There won't be a beta release?

**Comment by iJeys** [comment]
> There won't be a beta release?

do you want to create beta plots ? :)))

**Comment by madMAx43v3r** [comment]
the beta is internal and only one day planned

**Comment by iJeys** [comment]
> the beta is internal and only one day planned

Max while the plotter is in development, maybe there is an opportunity to publish the requirements here, for the plotter and video cards

**Comment by ddubick** [comment]
Lots of discussion on Discord. Much better place to have discussions like this than in a Github issue...

**Comment by strale92** [comment]
@madMAx43v3r
which is the minimum hardware for the next generation of plotter?

**Comment by Mircoxi** [comment]
> Lots of discussion on Discord. Much better place to have discussions like this than in a Github issue...

Gating knowledge and information behind joining a Discord server is bad in all circumstances - not least because not everyone *has* Discord. It's better to post updates in a public place for everyone.

**Comment by madMAx43v3r** [comment]
>  which is the minimum hardware for the next generation of plotter?

Same as before, plus there is a CUDA disk plotting mode now. The new format will only be supported with the CUDA plotter initially.

**Comment by QuintLeo** [comment]
Nice to have lots of discussion on Discord - but I for one have ZERO access to it at all.

**Comment by Kruger1981** [comment]
I had a look on the discord channel and yeah, lots of talk but nothing really about next version of plotter nor gigahorse...
Did I missed it?

**Comment by madMAx43v3r** [comment]
not released yet, just testing the farmer myself

**Comment by psychonz** [comment]
Hey @madMAx43v3r - like all the others here, I'm keenly awaiting a new release. But I'm happy to already have the amazing warez that you are already providing. Thanks so much for the good value solution =) Just wanted to say that. I'm sure your next release will be worth the wait. Peace.

**Comment by Kruger1981** [comment]
Hello @madMAx43v3r  just wondering if I should start moving my farmer to the more capable machine in preparation for the new C plots or leave it for now.
Asking as I can only do this over the weekends, and wondering if you have an ETA on the new toys for us :)
And as everyone else on here, really appreciate all that you do.

**Comment by madMAx43v3r** [comment]
Gonna be released by the weekend yes

**Comment by iJeys** [comment]
> Gonna be released by the weekend yes

I have plotters already ready)))
